pandas/tests/reshape/test_pivot.py

```diff
@@ -116,7 +116,9 @@ def test_pivot_table_dropna_categoricals(self):
 
         result_false = df.pivot_table(index='B', columns='A', values='C',
                                       dropna=False)
-        expected_columns = Series(['a', 'b', 'c', 'd'], name='A')
+        expected_columns = (
```

Comment:
I wanted to flag this change in the tests. AFAICT, this is actually the desired behavior? Looking for issues about this now.